css/style.css

```diff
@@ -0,0 +1,17 @@
+img {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
+  }
+  .title_paragraph{
+    display: block;
+    text-align: center;
+    background-color: rgb(233, 225, 225);
+  }
+  #tribute-info{
+    display: block;
+    text-align: center;
+    background-color: rgb(233, 225, 225);;
+  }
+  
```

Comment:
It is better to use flexbox and grid for layout of your Page it will make your website looks more good and responsive